sunpy/util/io.py

```diff
@@ -95,6 +127,27 @@ def is_url(obj):
         return False
 
 
+def is_uri(obj):
```

Comment:
Maybe I missed it, are there unit tests for this function?

Is there even a unit test collection for this file?!

Reply:
There's not an explicit test for this function but it does get called within `test_read_file_fits_uri`, `test_uri_pattern`, and `test_map_list_uri` in `test_map_factory.py` and `test_from_uri` in `test_timeseries_factory.py`.

Reply:
Thanks for detailing that.